tools/cli/commands/docs.js

```diff
@@ -38,16 +34,7 @@ export function docsDefine( yargs ) {
  * @param {argv} argv - the arguments passed.
  */
 export async function docsCli( argv ) {
-	let paths;
-	if ( 'all' === argv.project ) {
-		// "All" is a keyword for Jetpack plus packages.
-
-		paths = [ path.resolve( './projects/plugins/jetpack' ), path.resolve( './projects/packages' ) ];
-	} else {
-		paths = [ path.resolve( `./projects/plugins/${ argv.project }` ) ];
-	}
-
-	const parser_options = [ path.resolve( './tools/cli/helpers/doc-parser/runner.php' ), ...paths ];
+	const parser_options = [ path.resolve( './tools/cli/helpers/doc-parser/runner.php' ), argv.path ];
```

Comment:
Probably should add a check before this line to verify `argv.path` isn't empty, and `exit( 1 )` if it is. Otherwise we get a PHP fatal.